components/main.py

```
@@ -84,8 +84,13 @@ def worker(acc: Account) -> None:
         list(ex.map(worker, config.accounts))
 
     if not errors.empty():
-        reasons = "\n".join(list(errors.queue))
-        raise RuntimeError(f"Connectivity test failed for some accounts:\n{reasons}")
+        reason_lines: List[str] = []
+        while not errors.empty():
+            try:
+                reason_lines.append(errors.get_nowait())
+            except Exception:
+                break
+        raise RuntimeError(f"Connectivity test failed for some accounts:\n" + "\n".join(reason_lines))
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)